claimManagement/src/main/java/org/openimis/imisclaims/ClaimActivity.java

```diff
@@ -535,6 +540,81 @@ private void ClearForm(){
         etClaimAdmin.requestFocus();
     }
 
+    private void fillForm(JSONObject obj)
+    {
+        try {
+            String newClaimNumber = "@" + obj.getString("claim_number");
```

Comment:
The '@' prefix is configured in Web.config in WebApp. We should be able to change easily this prefix. Nepal put 're' as prefix. Please add this in strings.xml and take the value from there.

Reply:
I replaced it with a resource, so now i can be easily changed.